src/sas/qtgui/Utilities/ModelEditors/ReparamEditor/ReparameterizationEditor.py

```diff
@@ -199,8 +199,14 @@ def editSelected(self):
         
         # Find the parameter item by using param_to_open and format as a dictionary
         param_properties = self.getParamProperties(param_to_open)
-        param_properties['highlighted_property'] = highlighted_property  # TODO: Which property the cursor will start on
+        param_properties['highlighted_property'] = highlighted_property
         self.onAddParam()
+        # Populate the editor with the parameter values
+        if self.param_editor:
```

Comment:
Not clear to me that self.onAddParam should be called here if we are meant to just be editing an exisiting paramewter? In fact testing the installer does do something hybrid: it opens a parameter editor Window which is now populated with the values from the highlighted parameter but the result is to add all those parameters as a sub parameter of the same name!  In other words, if I create a parameter named Re and enter values then come back to edit it, the new parameter box will now show the old settings plus a NEW setting, named Re which itself has a bunch of unnamed settings but that contain the new values.

Reply:
@butlerpd, `self.onAddParam` is probably a misnomer. The method generates the param editor window and not much else. I am looking into the other issue of creating a separate tree inside the original tree.